packages/office-ui-fabric-react/src/components/Callout/Callout.types.ts

```diff
@@ -198,9 +198,18 @@ export interface ICalloutProps {
   theme?: ITheme;
 
   /**
-  * Optional styles for the component.
-  */
+   * Optional styles for the component.
+   */
   getStyles?: IStyleFunction<ICalloutContentStyleProps, ICalloutContentStyles>;
+  /**
+   * Whether or not the callout should be hidden rather than displayed.
```

Comment:
Better text:

If specified, renders the Callout in a hidden state. Use this flag, rather than rendering a callout conditionally based on visibility, to improve rendering performance when it becomes visible.

Reply:
also blank line between props